### PR TITLE
[FIX] fieldservice_agreement_helpdesk: Update Context Not Overwrite

### PR DESCRIPTION
**Pull request by osi-scampbell**
Update Context instead of overwriting it. Remove unnecessary xml.

### CODE REVIEW

File: helpdesk_fieldservice/views/fsm_order_views.xml [hunk 1]
```diff
@@ -8,10 +8,7 @@
         <field name="inherit_id" ref="fieldservice.fsm_order_form"/>
         <field name="arch" type="xml">
             <field name="template_id" position="after">
-                <field name="ticket_id" context="{'default_ticket_id': active_id}"/>
-            </field>
-            <field name="priority" position="attributes">
```
Comment (patrickrwilson) [inline helpdesk_fieldservice/views/fsm_order_views.xml]: @osi-scampbell are we losing the priority context here?

**Review by patrickrwilson — COMMENTED** [review]
@osi-scampbell Are we losing the priority context from the ticket to the fso?

**Comment by osi-scampbell** [comment]
@patrickrwilson We get rid of it here however it is already being set in helpdesk_fieldservice/models/helpdesk_ticket.py line 76

**Review by max3903 — APPROVED** [review]

**Comment by patrickrwilson** [comment]
> @patrickrwilson We get rid of it here however it is already being set in helpdesk_fieldservice/models/helpdesk_ticket.py line 76

got it, thanks

**Review by patrickrwilson — APPROVED** [review]

**Pull request merged**